tests/IntegrationTests/IntegrationTestService/Startup.cs

```diff
@@ -28,23 +28,32 @@ public void ConfigureServices(IServiceCollection services)
             KeyVaultSecretsProvider _keyVault = new KeyVaultSecretsProvider();
             string ccaSecret = _keyVault.GetSecret(TestConstants.OBOClientKeyVaultUri).Value;
 
-            var builder = services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
-                                  .AddMicrosoftIdentityWebApi(Configuration, subscribeToJwtBearerMiddlewareDiagnosticsEvents: true)
+            services.AddAuthentication()
```

Comment:
To have the default scheme, we could call `AddAuthentication(jwtBearerScheme: TestConstants.CustomJwtScheme)` just here too, and then leave the `WeatherForecastController ` controller as it was